includes/fbproduct.php

```diff
@@ -555,7 +556,7 @@ public function prepare_product( $retailer_id = null, $prepare_for_product_feed
 				'additional_image_urls' => array_slice( $image_urls, 1 ),
 				'url'                   => $product_url,
 				'category'              => $categories['categories'],
-				'brand'                 => $brand,
+				'brand'                 => SV_WC_Helper::str_truncate( $brand, 100 ),
```

Comment:
```suggestion
				'brand'                 => Framework\SV_WC_Helper::str_truncate( $brand, 100 ),
```

no need to strip tags here?

Reply:
Good catch! I tested and WordPress allows tags in the Site Title setting so it makes sense to strip the tags here as well.